README.md

```diff
@@ -1,92 +1,99 @@
-# [Lodex](http://lodex.inist.fr) &middot; [![Build Status](https://travis-ci.org/Inist-CNRS/lodex.svg?branch=master)](https://travis-ci.org/Inist-CNRS/lodex) [![bitHound Overall Score](https://cdn.rawgit.com/aleen42/badges/master/src/gitbook_1.svg)](https://lodex.gitbooks.io/lodex-user-documentation) [![Licence](https://img.shields.io/badge/licence-CeCILL%202.1-yellow.svg)](http://www.cecill.info)
+[![Lodex CI](https://github.com/Inist-CNRS/lodex/actions/workflows/lodex-ci.yml/badge.svg)](https://github.com/Inist-CNRS/lodex/actions/workflows/lodex-ci.yml)
+[![Licence](https://img.shields.io/badge/licence-CeCILL%202.1-yellow.svg)](http://www.cecill.info)
+[![Documentation](https://img.shields.io/badge/Documentation-f48022)](https://lodex.inist.fr/docs/)
+![Docker Image Version (latest semver)](https://img.shields.io/docker/v/inistcnrs/lodex)
```

Comment:
Strange: this image has a link (to <https://camo.githubusercontent.com/9b2a32c713dbd8b402938134e8cc03295a39b23ddb8e64b1eb1ed419d6068061/68747470733a2f2f696d672e736869656c64732e696f2f646f636b65722f762f696e697374636e72732f6c6f646578>), but I don't understand how.

Reply:
I forgot to add the link to docker hub...